js/app.js

```diff
@@ -198,36 +198,68 @@ function tagIsSelected(blogTag) {
 
 
 const snippetTypes = {
-    "media_embed": { "property": "media_item__text.value", "tag": "div", "format": "html" },
-    "code_snippet": { "property": "code", "tag": "pre", "format": "text" },
-    "aside_note": { "property": "note_text", "tag": "aside", "format": "html" }
+    "media_embed": { "property": "media_item__text.value", "tag": "div", "format": "html", "classname": "" },
+    "code_snippet": { "property": "code", "tag": "pre", "format": "text", "classname": "" },
+    "aside_note": { "property": "note_text", "tag": "aside", "format": "html", "classname": "" },
+    "quotation": { "property": "quote", "tag": "blockquote", "format": "html", "classname": "quotation" }
     };
+var elementSnippet;
```

Comment:
These script-scoped variables are a code smell. Equivalent to globals in VB, they are considered very bad practice (albeit sometimes undeservedly).

Reply:
The algorithm below does not need global access to `objSnippet`. It looks like it's only needed for the very last operation (loadElementSnippet) and could just be passed in. I think there are only two functions to pass elementSnippet to.